set_up/env_template.j2

```diff
@@ -0,0 +1,7 @@
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+TRELLO_BOARD_KEY = "{{boardKey}}"
+TRELLO_BOARD_TOKEN = "1ca79abd9c61a5acf80de332b064d51adcce3bef9b5241a958d8eb11e1890bae"
```

Comment:
The token should also be a variable (especially if you actually care about your Trello board)